lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -285,34 +286,15 @@ public boolean hasMatte() {
     return compositionLayer != null && compositionLayer.hasMatte();
   }
 
-  public boolean enableMergePathsForKitKatAndAbove() {
-    return enableMergePaths;
-  }
-
-  /**
-   * Enable this to get merge path support for devices running KitKat (19) and above.
-   * <p>
-   * Merge paths currently don't work if the the operand shape is entirely contained within the
-   * first shape. If you need to cut out one shape from another shape, use an even-odd fill type
-   * instead of using merge paths.
-   */
   public void enableMergePathsForKitKatAndAbove(boolean enable) {
```

Comment:
We will probably deprecate this method and incentivise use of the more generic enableFeatureFlag function. I was wondering if we pipe it in through here LottieDrawable, or do we do all of our flag setting through LottieComposition.

Reply:
Settled on LottieDrawable to allow different drawables different flags.